deepem/data/dataset/zettaset.py

```diff
@@ -70,15 +78,30 @@ def convert_array(arr: ArrayLike) -> np.ndarray:
 
         # Mask
         if key in sample.masks:
-            vol = sample.read_mask(key)[key]            
+            vol = sample.read_mask(key)[key]
             dset[name + "_mask"] = convert_array(vol).astype('uint8')
         else:
             dset[name + "_mask"] = np.ones_like(dset[name], dtype='uint8')
         print(f"{name + '_mask'}: {dset[name + '_mask'].shape}")
 
+        # applying padding
+        if zettaset_padding != (0, 0, 0):
+            widths = (
```

Comment:
This is always a source of confusion, but `zettaset_padding` is supposed to be in (x, y, z) order, right?
Both labels and their masks have been converted to (z, y, x) order in the above, so I guess the correct padding widths here should be:
```
widths = (
    (zettaset_padding[-1], zettaset_padding[-1]),
    (zettaset_padding[-2], zettaset_padding[-2]),
    (zettaset_padding[-3], zettaset_padding[-3]),
)
```

@nicholasturner1 Can you please confirm it?

Reply:
I made this argument use the same zyx convention as `fov`, `outputsz`, etc., so I think the current code should be correct.